docs/using.rst

```diff
@@ -188,6 +190,35 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant bi-directional folder sync
+==================================
+
+For basic usage please refer `Vagrant documentation. <https://www.vagrantup.com/docs/synced-folders/basic_usage.html>`_
+
+Vagrant bi-directional sync folder is a very useful feature in Vagrant as it is the simplest way to move files (e.g. code) between
```

Comment:
"Vagrant's synced folders are a very powerful feature providing a simple way to move files (e.g code) between host and Vagrant guest"